src/open_inwoner/accounts/views/registration.py

```diff
@@ -96,17 +100,25 @@ def get_context_data(self, **kwargs):
             else reverse("profile:registration_necessary")
         )
         try:
+            config = OpenIDConnectDigiDConfig.get_solo()
+            if config.enabled:
+                digid_url = reverse("digid_oidc:init")
+            else:
+                digid_url = reverse("digid:login")
             context["digit_url"] = (
-                furl(reverse("digid:login")).add({"next": necessary_fields_url}).url
+                furl(digid_url).add({"next": necessary_fields_url}).url
             )
         except:
```

Comment:
The try-except is present to catch `NoReverseMatch` errors when trying to `reverse("digid:login")` (because these URLs are added based on environment variables. I'll specifically catch that exception